src/thorin/primop.h

```diff
@@ -37,7 +37,12 @@ class PrimOp : public Def {
 protected:
     virtual uint64_t vhash() const;
     virtual bool equal(const PrimOp* other) const;
+#ifdef _MSC_VER
+    virtual const Def* vrebuild(World& to, Defs ops, const Type* type) const { return nullptr; } //  = 0;
+#else  // _MSC_VER
     virtual const Def* vrebuild(World& to, Defs ops, const Type* type) const = 0;
+#endif // _MSC_VER
+
     /// Is @p def the @p i^th result of a @p T @p PrimOp?
```

Comment:
I'm fine with just having the upper version which just returns ```nullptr```. No need for a ```#ifdef``` here. But plz remove the parameter names, otherwise you'll get a ton of unused parameter warnings.